src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/DeleteSpecificCommentAuthorizationHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿#nullable enable
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
+{
+    public class DeleteSpecificCommentRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class DeleteSpecificCommentAuthorizationHandler
```

Comment:
As this is a new handler that needs to be supplied via DI, you'll need to register it in `StartupSecurityConfig.ConfigureResourceBasedAuthorization()` or you'll get a null pointer error when a Service tries to use it.